custom/m4change/templates/m4change/fields/daterange.html

```diff
@@ -31,7 +31,7 @@
 {% endblock %}
 {% block filter_js %}
     <script type="text/javascript">
-        $.getScript("{% static 'm4change/javascripts/daterangepicker.js' %}", function () {
+        $.getScript("{% static 'm4change/js/daterangepicker.js' %}", function () {
```

Comment:
might be missing something but it doesn't look like this file moves in this PR to me.